10-setup-htcondor.sh

```diff
@@ -101,4 +123,8 @@ mkdir -p `condor_config_val SPOOL`
 mkdir -p `condor_config_val SEC_CREDENTIAL_DIRECTORY`
 chmod 600 `condor_config_val SEC_CREDENTIAL_DIRECTORY`
 
+echo
+echo "Will use the following token:"
```

Comment:
```suggestion
echo "Will use the following token(s):"
```

A single file may actually contain multiple new-line separated tokens

Reply:
Ah ok!